examples/command/portals/snowflake/example-5/README.md

```diff
@@ -0,0 +1,290 @@
+
+```
+
+## Build and push Python application Image
+
+
+```sh
+cd snowflake_cdc_publisher
+
+# Use the value of the repository_url from SHOW IMAGE REPOSITORIES command
+docker login <repository_url>
+docker build --rm --platform linux/amd64 -t <repository_url>/snowflake_cdc_kafka_bridge .
+docker push <repository_url>/snowflake_cdc_kafka_bridge
+
+cd -
+```
+
+## Deploy Snowflake CDC Publisher with Ockam in Snowpark Container services 
+
+> [!IMPORTANT]
+> Replace `TODO` values in `VALUE_LIST` with the output of `ockam project show --jq .egress_allow_list` command in previous step.
+
+```sh
+#Example
+VALUE_LIST = ("k8s-XXX.amazonaws.com:4XXX","k8s-XXX.amazonaws.com:4XXX");
+```
+
+> [!IMPORTANT]
+> Replace `<OCKAM_ENROLLMENT_TICKET>` with contents of `inlet.ticket` generated in previous step
+
+```sql
+
+USE ROLE CDC_TEST_ROLE;
+USE DATABASE CDC_TEST_DB;
+USE WAREHOUSE CDC_TEST_WH;
+USE SCHEMA CDC_TEST_SCHEMA;
+
+CREATE OR REPLACE NETWORK RULE CDC_TEST_OSCP_OUT
+TYPE = 'HOST_PORT' MODE= 'EGRESS'
+VALUE_LIST = ('ocsp.snowflakecomputing.com:80');
+
+
+-- Update VALUE_LIST with ockam egress details
+CREATE NETWORK RULE CDC_TEST_OCKAM_OUT TYPE = 'HOST_PORT' MODE = 'EGRESS'
+VALUE_LIST = ('TODO:TODO', 'TODO:TODO');
+
+
+-- Create access integration
+USE ROLE ACCOUNTADMIN;
+
+CREATE OR REPLACE EXTERNAL ACCESS INTEGRATION CDC_TEST_EXTERNAL_ACCESS_INT
+ALLOWED_NETWORK_RULES = (CDC_TEST_OSCP_OUT, CDC_TEST_OCKAM_OUT)
+ENABLED = true;
+
+GRANT USAGE ON INTEGRATION CDC_TEST_EXTERNAL_ACCESS_INT TO ROLE CDC_TEST_ROLE;
+
+-- Create service
+USE ROLE CDC_TEST_ROLE;
+
+DROP SERVICE IF EXISTS SNOWFLAKE_CDC_KAFKA_BRIDGE;
+
+CREATE SERVICE SNOWFLAKE_CDC_KAFKA_BRIDGE
+  IN COMPUTE POOL CDC_TEST_CP
+  FROM SPECIFICATION
+$$
+    spec:
+      containers:
+      - name: publisher
+        image: /cdc_test_db/cdc_test_schema/cdc_test_repo/snowflake_cdc_kafka_bridge
+        env:
+          STREAM_NAME: CDC_TEST_DB.CDC_TEST_SCHEMA.CDC_TEST_TABLE_STREAM
+          KAFKA_BOOTSTRAP_SERVERS: 127.0.0.1:9092
+          KAFKA_TOPIC_NAME: test-topic
+          SNOWFLAKE_WAREHOUSE: CDC_TEST_WH
+          JOB_SUCCESS_SLEEP_TIME: 30
+          JOB_ERROR_SLEEP_TIME: 60
+          OCKAM_DISABLE_UPGRADE_CHECK: true
+          OCKAM_OPENTELEMETRY_EXPORT: false
+          ENROLLMENT_TICKET: "<OCKAM_ENROLLMENT_TICKET>"
+$$
+EXTERNAL_ACCESS_INTEGRATIONS = (CDC_TEST_EXTERNAL_ACCESS_INT)
+MIN_INSTANCES=1
+MAX_INSTANCES=1;
+
+SHOW SERVICES;
+SELECT SYSTEM$GET_SERVICE_STATUS('SNOWFLAKE_CDC_KAFKA_BRIDGE');
+DESCRIBE SERVICE SNOWFLAKE_CDC_KAFKA_BRIDGE;
+CALL SYSTEM$GET_SERVICE_LOGS('SNOWFLAKE_CDC_KAFKA_BRIDGE', '0', 'publisher', 1000);
+
+```
+
+## Verify
+
+```sql
+-- Insert records
+INSERT INTO CDC_TEST_TABLE
+SELECT UUID_STRING(), randstr(255, RANDOM())
+FROM TABLE(GENERATOR(ROWCOUNT => 100));
+
+-- Verify Test Table and CDC Table
+SELECT * FROM CDC_TEST_TABLE;
+SELECT * FROM CDC_TEST_TABLE_STREAM;
+
+-- Publisher checks for changes every 30 seconds. Looks at logs
```

Comment:
In the comments we could add an example of what an expected log line looks like.